### PR TITLE
FEAT: Support new model CogVLM

### PR DESCRIPTION
**Pull request by amumu96**
image

### CODE REVIEW

File: xinference/model/llm/pytorch/cogvlm2.py [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+                return text, None
+            elif len(images) == 1:
+                return text, images
+            else:
+                raise RuntimeError(
+                    "Only one image per message is supported by CogVLM2."
+                )
+        return content, None
+
+    def _history_content_to_intern(
```
Comment (codingl2k1) [inline xinference/model/llm/pytorch/cogvlm2.py:135]: Is this method useful for CogVLM? It shows that `_history_content_to_intern`

Comment (amumu96) [reply xinference/model/llm/pytorch/cogvlm2.py]: yes, just forget to rename func,  ill fix it

**Review by qinxuye — APPROVED** [review]
LGTM

**Comment by fire17** [comment]
can you guys add CogVLM2 to the BrillAI platform?
I want to test its pixel coordinate functionality - ie
return the location of the button named "retry" in [[x0,y0,x1,y1]] format
it should return the bounding box in values between 1-1000 in reference to the image,
locating / pinpointing vlm is not something i have seen yet but cogvlm2 seems to be able to do this
Thanks :) and all the best

**Pull request merged**